rtos/TARGET_CORTEX/mbed_rtx_conf.h

```diff
@@ -45,10 +45,18 @@
 #else
 #define EXTRA_IDLE_STACK 0
 #endif
+
+// Increase the idle thread stack size when debug is enabled
+#if defined(MBED_DEBUG)
+#define EXTRA_IDLE_STACK_DEBUG MBED_CONF_RTOS_IDLE_THREAD_STACK_SIZE_DEBUG_EXTRA
+#else
+#define EXTRA_IDLE_STACK_DEBUG 0
+#endif
+
 #ifdef MBED_CONF_APP_IDLE_THREAD_STACK_SIZE
-#define OS_IDLE_THREAD_STACK_SIZE   (MBED_CONF_APP_IDLE_THREAD_STACK_SIZE + EXTRA_IDLE_STACK)
+#define OS_IDLE_THREAD_STACK_SIZE   MBED_CONF_APP_IDLE_THREAD_STACK_SIZE
```

Comment:
I changed this line,
as I think that different calculation for idle size value depending on tickless and/or debug
is needed only if the user doesn't specify explicitly the needed and expected size.

@c1728p9